### PR TITLE
EMBOSS alignment parser, resurrected

### PR DESCRIPTION
**Pull request by mdehoon**
Resurrected from #3657 

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``pre-commit`` locally,
and understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

\

### CODE REVIEW

File: Bio/Align/__init__.py [hunk 1]
```diff
@@ -938,7 +938,7 @@ class Alignment:
     """
 
     @classmethod
-    def _infer_coordinates(cls, lines):
+    def infer_coordinates(cls, lines):
         """Infer the coordinates from a printed alignment."""
```
Comment (peterjc) [inline Bio/Align/__init__.py:942]: What this method does, and what it returns, are not clear from the one line docstring.

That was OK when it was private, but we should aim higher for a public API.

File: Bio/Align/emboss.py [hunk 2]
```diff
@@ -0,0 +1,217 @@
+# Copyright 2008-2016 by Peter Cock.  All rights reserved.
+#
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
+"""Bio.AlignIO support for "emboss" alignment output from EMBOSS tools.
```
Comment (peterjc) [inline Bio/Align/emboss.py:7]: This is clearly based on ``Bio/AlignIO/EmbossIO.py`` but this docstring needs tweaking slightly now, both here and a few lines down (although that hypothetical new high level API does not exist yet).

File: Bio/Align/__init__.py (hunk 1 above)
Comment (peterjc) [reply Bio/Align/__init__.py]: That's much better, thank you

**Review by peterjc — APPROVED** [review]

**Comment by mdehoon** [comment]
Thank you

**Pull request merged**